### ISSUE TITLE
Is this still worked?

### ISSUE THREAD

**Issue by riskyprsty** [miscord/miscord#462]
I have an error after completed the localhost setup. This is the error message
![Screenshot_2020-07-03-15-56-12](https://user-images.githubusercontent.com/66415817/86452466-8847f900-bd46-11ea-8bdf-0daa9806c0e2.png)

**Comment by ptrcnull** [comment]
Hey, that error message doesn't give me much info. Does anything appear in the console?

**Comment by riskyprsty** [comment]
Here's the error message in console
![Screenshot_2020-07-04-07-50-43](https://user-images.githubusercontent.com/66415817/86502283-b1f73380-bdcb-11ea-8e40-41c607b64895.png)
![Screenshot_2020-07-04-07-50-31](https://user-images.githubusercontent.com/66415817/86502286-b7547e00-bdcb-11ea-8bf9-6773683262d3.png)

**Comment by ArtemMkV** [comment]
i have exactly the same error. but the post from riskyprsty is very unreadable so i will paste it. 

```
 SUCCESS  02:37:45  ›  Worker process 59660 started.
  START   02:37:45  ›  Compressing old logs...
 SUCCESS  02:37:45  ›  All old logs compressed, starting Miscord
  INFO    02:37:45 FileConfig ›  Loading config from /Users/mkv/Library/Application Support/Miscord/config.json
  START   02:37:45  ›  Launching Miscord v5.0.4
  START   02:37:45 login ›  Logging in...
  INFO    02:37:45 login › logLevel 'info'
  START   02:37:45 discord › login ›  Logging in to Discord...
 SUCCESS  02:37:46 discord › login ›  Logged in to Discord
  START   02:37:46 messenger › login ›  Logging in to Facebook...
 SUCCESS  02:37:46 messenger › login ›  Logged in to Facebook
 SUCCESS  02:37:46 login ›  Logged in
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
AssertionError [ERR_ASSERTION]: Method 'GET' already declared for route '/connections/'
    at Router._insert (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:286:9)
    at Router._on (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:188:8)
    at Router.on (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:73:8)
    at Object.afterRouteAdded (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/fastify/lib/route.js:232:16)
    at /Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/fastify/lib/route.js:163:31
    at Object._encapsulateThreeParam (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:468:7)
    at Boot.timeoutCall (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:379:5)
    at Boot.callWithCbOrNextTick (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:360:19)
    at Boot._after (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:235:26)
    at Plugin.exec (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/plugin.js:89:17)
  ERROR   02:37:46 errorHandler ›  AssertionError [ERR_ASSERTION]: Method 'GET' already declared for route '/connections/'
  ERROR   02:37:46 errorHandler ›      at Router._insert (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:286:9)
  ERROR   02:37:46 errorHandler ›      at Router._on (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:188:8)
  ERROR   02:37:46 errorHandler ›      at Router.on (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/find-my-way/index.js:73:8)
  ERROR   02:37:46 errorHandler ›      at Object.afterRouteAdded (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/fastify/lib/route.js:232:16)
  ERROR   02:37:46 errorHandler ›      at /Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/fastify/lib/route.js:163:31
  ERROR   02:37:46 errorHandler ›      at Object._encapsulateThreeParam (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:468:7)
  ERROR   02:37:46 errorHandler ›      at Boot.timeoutCall (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:379:5)
  ERROR   02:37:46 errorHandler ›      at Boot.callWithCbOrNextTick (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:360:19)
  ERROR   02:37:46 errorHandler ›      at Boot._after (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/boot.js:235:26)
  ERROR   02:37:46 errorHandler ›      at Plugin.exec (/Users/mkv/.nvm/versions/node/v14.9.0/lib/node_modules/miscord/node_modules/avvio/plugin.js:89:17) {
  ERROR   02:37:46 errorHandler ›    generatedMessage: false,
  ERROR   02:37:46 errorHandler ›    code: 'ERR_ASSERTION',
  ERROR   02:37:46 errorHandler ›    actual: false,
  ERROR   02:37:46 errorHandler ›    expected: true,
  ERROR   02:37:46 errorHandler ›    operator: '=='
  ERROR   02:37:46 errorHandler ›  }
  INFO    02:37:47  ›  Worker process 59660 died (2, null).
 SUCCESS  02:37:47  ›  Worker process 59662 started.
  START   02:37:47  ›  Compressing old logs...
 SUCCESS  02:37:47  ›  All old logs compressed, starting Miscord
  INFO    02:37:47 FileConfig ›  Loading config from /Users/mkv/Library/Application Support/Miscord/config.json
  START   02:37:47  ›  Launching Miscord v5.0.4
  START   02:37:47 login ›  Logging in...
  INFO    02:37:47 login › logLevel 'info'
  START   02:37:47 discord › login ›  Logging in to Discord...
 SUCCESS  02:37:51 discord › login ›  Logged in to Discord
  START   02:37:51 messenger › login ›  Logging in to Facebook...
 SUCCESS  02:37:51 messenger › login ›  Logged in to Facebook
 SUCCESS  02:37:51 login ›  Logged in
...... looping without end .....
```

**Comment by ptrcnull** [comment]
~~Duplicate of #470,~~ err, nope, wrong issue.
Nontheless, will be fixed once the change is merged to stable.

**Comment by csandman** [comment]
Was this fixed?  I am still running into this error in some deployments

**Comment by ptrcnull** [comment]
Miscord itself doesn't work because of a different issue, but it caused this bugfix to not get merged yet.

**Comment by csandman** [comment]
@ptrcnull Interesting, because I managed to get a version set up last week that appears to be fully functional.  In fact, I helped an admin of the New Hampshire Pokemon Go discord server get it back up and running for the 14 different Messenger bridges they had previously had working there.

All I did was have them build the project from source and it's running without issue!

**Comment by racinmat** [comment]
Hi, I'm also getting this issue when I build it on dev branch.
I can't use the last stable version because of #474 
 @csandman  how did you manage to run it without this error, which commit did you build?
